packages/contracts/src/score.schemas.ts

```diff
@@ -0,0 +1,75 @@
+import { Static, Type } from '@sinclair/typebox';
+export const WinnerIdField = Type.Number();
+export const LoserIdField = Type.Number();
```

Comment:
```suggestion
import { IdField } from './user.schemas';

export const WinnerIdField = IdField;
export const LoserIdField = IdField;
```

this way we can control the IdField specifications from just 1 place in `user.schemas.ts`
sure the id will probably not change too much, or will not have many constraints but still for consistency I think it's nice?

maybe some other ones that could be similarily generalized, but not crazy important